inst/python/abbreviateStackTrace.py

```diff
@@ -0,0 +1,12 @@
+EXCEPTION_MESSAGE_BOUNDARY='exception-message-boundary'
+
+def abbreviateStackTrace(function):
+    exceptionToRaise = None
+    try:
+        return function()
+    except Exception as e:
+        exceptionToRaise = e     
+    # We do this to suppress the automatic exception chaining that occurs when rethrowing
+    # with an 'except' block
+    if exceptionToRaise is not None:
+        raise Exception(EXCEPTION_MESSAGE_BOUNDARY+str(exceptionToRaise)+EXCEPTION_MESSAGE_BOUNDARY)
```

Comment:
This is cool, can you provide an example of the output of this function?  Does this basically remove the chained stack traces that you sometimes get from Python and only output the most recent exception?

Because if so: TIL - that you can do this.

Reply:
I thought this was useful as well, but can't take credit for it - I brought it over from the `PythonEmbedInR` repo. Happy to whip up an example if you're still interested.

Reply:
No need - I think I understand the context.